src/bls/bls.h

```diff
@@ -148,7 +149,7 @@ class CBLSWrapper
     }
 
     //TODO: Add version with argument (bool specificLegacyScheme)
-    bool SetHexStr(const std::string& str)
+    bool SetHexStr(const std::string& str, const bool specificLegacyScheme)
```

Comment:
I think it is better to use `enum class` instead bool for better code reability.
Otherwise it is more difficult to read code such as "SetHexStr(str, true)" without using IDE's hints.

Reply:
Also this bool can be extra pain in case if it would be once 3rd version of BLS

Reply:
Right, there are two other schemes in the BLS library.